### PR TITLE
2268 refractor radio buttons

### PR DESCRIPTION
**Pull request by AlexSofft**

### CODE REVIEW

**Comment by mf-code** [comment]
[5935] - CQ5sx2GUtUXB

**Comment by mf-code** [comment]
[5935] - pyzvhMv76sLo

**Comment by mf-code** [comment]
[5935] - RTnMst0QxK82

**Comment by mf-code** [comment]
[5935] - dyc0J4hoJDMO

**Comment by mf-code** [comment]
[5935] - 2sT0b6LZCRbO

**Comment by mf-code** [comment]
[5935] - bE4p95A0IiaC

**Comment by mf-code** [comment]
[5935] - 7m6aRj5sG7H9

**Comment by mf-code** [comment]
[5935] - gCMclhlW7Yag

**Comment by mf-code** [comment]
[5935] - sTfGl0H1VT4R

**Comment by mf-code** [comment]
[5935] - nMbLdbKhS9nb

**Comment by mf-code** [comment]
[5935] - yz7w8aAQnYvs

**Comment by mf-code** [comment]
[5935] - wdcgD3KO7Pk0

**Comment by mf-code** [comment]
[5938] - ACtzuQY383Sr

**Comment by mf-code** [comment]
[5938] - YtFTj9DdnORq

**Comment by mf-code** [comment]
[5938] - hjwW2SWn2c3S

**Comment by mf-code** [comment]
[5938] - M2fHu757UejG

**Comment by mf-code** [comment]
[5938] - mxIw8zIFLGMO

**Comment by mf-code** [comment]
[5938] - lq8lkwdPYMp6

**Comment by mf-code** [comment]
[5938] - FTwf1hakGbpH

**Comment by mf-code** [comment]
[5938] - iF3mZ4GpL1z0

**Comment by mf-code** [comment]
[5938] - wGd6JXDMASy5

**Comment by mf-code** [comment]
[5938] - mb3DqhtZbpxZ

**Comment by mf-code** [comment]
[5938] - A2SEFE93FFrY

**Comment by mf-code** [comment]
[5938] - acvOlz0lrRTO

**Comment by mf-code** [comment]
[5944] - COBnGYk4ntv8

**Comment by mf-code** [comment]
[5944] - 8tZeMmkhbMfW

**Comment by mf-code** [comment]
[5944] - 9JdaoavrjVrZ

**Comment by mf-code** [comment]
[5944] - mo7kur5gJV8a

**Comment by mf-code** [comment]
[5944] - Vip9TDCbTS4a

**Comment by mf-code** [comment]
[5944] - Fikc0e6jA5vJ

**Comment by mf-code** [comment]
[5944] - 2ll8lvz9Kuxw

**Comment by mf-code** [comment]
[5944] - L83gsgo6pXov

**Comment by mf-code** [comment]
[5944] - LFZ2OwnZ93dw

**Comment by mf-code** [comment]
[5944] - ZFDSn1uubZFd

**Comment by mf-code** [comment]
[5944] - ELBNzY0nYKk1

**Comment by mf-code** [comment]
[5944] - wtoDkhLcCx7p

**Comment by mf-code** [comment]
[5944] - 1CuEcDrhdWGt

**Comment by mf-code** [comment]
[5944] - B1gGx98EYMNf

**Comment by mf-code** [comment]
[5944] - 9ZuNxJNjprjP

**Comment by mf-code** [comment]
[5944] - JWrAcxIe8hRT

**Comment by mf-code** [comment]
[5944] - openjdk8 - Allure report on Netlify: https://5f48cb7ea1413d676d3b5784--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5944] - openjdk10 - Allure report on Netlify: https://5f48cc4875f3d2ce341b738a--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5961] - FhFPPAB1BCHr

**Comment by mf-code** [comment]
[5961] - z8frfyMz3a2g

**Comment by mf-code** [comment]
[5961] - r4tgjVkI1DsT

**Comment by mf-code** [comment]
[5961] - 6CVY2zzNhPcg

**Comment by mf-code** [comment]
[5961] - rlG0r5eBxBz3

**Comment by mf-code** [comment]
[5961] - pWYtWM4mW64J

**Comment by mf-code** [comment]
[5961] - ezFOCNSJd3IJ

**Comment by mf-code** [comment]
[5961] - Vz9w7wxt7pXy

**Comment by mf-code** [comment]
[5961] - ZhkN7LW7iMZV

**Comment by mf-code** [comment]
[5961] - E0Tc5U4JJBrU

**Comment by mf-code** [comment]
[5961] - 7wqhSkF7tTZq

**Comment by mf-code** [comment]
[5961] - P2j8J9cUi3R0

**Comment by mf-code** [comment]
[5961] - rdIUkC1kqHyw

**Comment by mf-code** [comment]
[5961] - znGrF8OZTzjC

**Comment by mf-code** [comment]
[5961] - t6isKylQ3hVO

**Comment by mf-code** [comment]
[5961] - n4E37zdQrFPm

**Comment by mf-code** [comment]
[5961] - Z0saa7nxMllU

**Comment by mf-code** [comment]
[5961] - zFeHEeAkswoF

**Comment by mf-code** [comment]
[5961] - openjdk8 - Allure report on Netlify: https://5f49294dc17d21728b6c36e4--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5961] - openjdk10 - Allure report on Netlify: https://5f492aded786b88503e6bca9--musing-swartz-6fc060.netlify.app

File: jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/RadioButtonSection.java [hunk 1]
```diff
@@ -1,36 +1,17 @@
...
 public class RadioButtonSection {
-    @Css("#basic-radio-group")
-    public static RadioButtons basicRadioButtons;
-
-    @UI("#radio-option-one")
-    public static Button firstBasicRadioButton;
 
-    @UI("#radio-option-two")
-    public static Button secondBasicRadioButton;
+    @UI("#basic-radio-group")
```
Comment (gbuh) [inline jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/RadioButtonSection.java:9]: The annotation @UI("#basic-radio-group") is not required here.
The "smart locator" is working properly.

**Comment by mf-code** [comment]
[5972] - mn89exeX2Om2

**Comment by mf-code** [comment]
[5972] - Rmooy7zZleD5

**Comment by mf-code** [comment]
[5972] - yLAkKbX31QKI

**Comment by mf-code** [comment]
[5972] - qAB76zxjBJcr

**Comment by mf-code** [comment]
[5972] - yj49eQnvuPSV

**Comment by mf-code** [comment]
[5972] - nKaiQofoarQb

**Comment by mf-code** [comment]
[5972] - fth1NRjpJytX

**Comment by mf-code** [comment]
[5972] - 67AnbVrjV2ci

**Comment by mf-code** [comment]
[5972] - g72j0ieqWhYy

**Comment by mf-code** [comment]
[5972] - wnmdKvG9LwxG

**Comment by mf-code** [comment]
[5972] - nWHPdMJSDCCO

**Comment by mf-code** [comment]
[5972] - oRVbXkJQEmKV

**Comment by mf-code** [comment]
[5972] - EWgof1dDfm3o

**Comment by mf-code** [comment]
[5972] - fczLPiRHCFwa

**Comment by mf-code** [comment]
[5972] - TyiX1UnpEM34

**Comment by mf-code** [comment]
[5972] - SOJ9IDItqxMK

**Comment by mf-code** [comment]
[5972] - HmwyxdckHwV4

**Comment by mf-code** [comment]
[5972] - Ylgkzne1QHgN

**Comment by mf-code** [comment]
[5972] - openjdk8 - Allure report on Netlify: https://5f4cee09d5727224027074ab--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5972] - openjdk10 - Allure report on Netlify: https://5f4cef8712091d2bdbff1ea9--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5972] -

**Comment by mf-code** [comment]
[5972] -

**Comment by mf-code** [comment]
[5972] - pRFikUTbx8jZ

**Comment by mf-code** [comment]
[5972] - DhssFZa2HyOZ

**Comment by mf-code** [comment]
[5972] -

**Comment by mf-code** [comment]
[5972] - nIo5wxDBApW2

**Comment by mf-code** [comment]
[5972] - tvsISfw3qYV4

**Comment by mf-code** [comment]
[5972] - se4XXE9f23rR

**Comment by mf-code** [comment]
[5972] - IM7JM0F7PaPu

**Comment by mf-code** [comment]
[5972] - SxsLiiDIuFhv

**Comment by mf-code** [comment]
[5972] - 47GB9Gs3SYYR

**Comment by mf-code** [comment]
[5972] - 7WreclgKfpEt

**Comment by mf-code** [comment]
[5972] - DFukqFt4BMsw

**Comment by mf-code** [comment]
[5972] - qXAsXSIQ7biL

**Comment by mf-code** [comment]
[5972] - gEnJPVYm9BO8

**Comment by mf-code** [comment]
[5972] - nyYvIq5bbLqS

**Comment by mf-code** [comment]
[5972] - KGp2O9Z5vUk8

**Comment by mf-code** [comment]
[5972] - gvIJUmMcZDTB

**Comment by mf-code** [comment]
[5972] - openjdk8 - Allure report on Netlify: https://5f4e9687a1324d008e2f663b--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5972] - openjdk10 - Allure report on Netlify: https://5f4e97a5933b0200a29dc980--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6004] - uMaQ86U9CSqG

**Comment by mf-code** [comment]
[6004] - qmbR0Sy6Jj9m

**Comment by mf-code** [comment]
[6004] - z4gLbzEvrDmT

**Comment by mf-code** [comment]
[6004] - XoXEVnDkemwB

**Comment by mf-code** [comment]
[6004] - JQ1Rtp3HPjNK

**Comment by mf-code** [comment]
[6004] - RN7r6CMU4OkG

**Comment by mf-code** [comment]
[6004] - nDvvh7oztX4B

**Comment by mf-code** [comment]
[6004] - QfU3fGxJC2Ou

**Comment by mf-code** [comment]
[6004] - qoseBp4jZMHB

**Comment by mf-code** [comment]
[6004] - iCxnPPP5Aen7

**Comment by mf-code** [comment]
[6004] - F5yQ0t0Q3z3Z

**Comment by mf-code** [comment]
[6004] - gB80Wq7zlrPn

**Comment by mf-code** [comment]
[6004] - 2uBM0iR1eVbO

**Comment by mf-code** [comment]
[6004] - OAo3Hcm5UAPj

**Comment by mf-code** [comment]
[6004] - tSYy8ANRVXHY

**Comment by mf-code** [comment]
[6004] - EWeKC2wbARZc

**Comment by mf-code** [comment]
[6004] - TwXYU5asQuCT

**Comment by mf-code** [comment]
[6004] - ii9qAzj2m7Rn

**Comment by mf-code** [comment]
[6004] - openjdk8 - Allure report on Netlify: https://5f4fab5c9370d11c0f5278af--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6004] - openjdk10 - Allure report on Netlify: https://5f4fac483c5e2918c6f47c34--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6004] - kaQdjW5sHEXN

**Comment by mf-code** [comment]
[6004] - aFOIWNBBEqzN

**Comment by mf-code** [comment]
[6004] - 4kPnxzoSMpoq

**Comment by mf-code** [comment]
[6004] - 6i4nNbLIDQsO

**Comment by mf-code** [comment]
[6004] - 5XXUCLw8zJhv

**Comment by mf-code** [comment]
[6004] - NcCe1SR8b9Fz

**Comment by mf-code** [comment]
[6004] - 1uE5YlZGTlPd

**Comment by mf-code** [comment]
[6004] - vyM79gdlYS8d

**Comment by mf-code** [comment]
[6004] - 8osZ93t6b2uC

**Comment by mf-code** [comment]
[6004] - 7RyO3h6fL2jy

**Comment by mf-code** [comment]
[6004] - nrV93yvtfMmW

**Comment by mf-code** [comment]
[6004] - s3PG8DdAlq94

**Comment by mf-code** [comment]
[6004] - zY7ukUSfZljl

**Comment by mf-code** [comment]
[6004] - ytvUoZxiHWPm

**Comment by mf-code** [comment]
[6004] - blvX8BtTiXAZ

**Comment by mf-code** [comment]
[6004] - zJW5PTohFcbO

**Comment by mf-code** [comment]
[6004] - HJkaLdAFw5FL

**Comment by mf-code** [comment]
[6004] - hyFDoFK59rXf

**Comment by mf-code** [comment]
[6004] - openjdk10 - Allure report on Netlify: https://5f4ff00c49c0c62d75ba8cca--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[6011] - ztQ9S8FCwhTv

**Comment by mf-code** [comment]
[6011] - P5oLGAlZEZgc

**Comment by mf-code** [comment]
[6011] - WjtLxglSOVRh

**Comment by mf-code** [comment]
[6011] - Z62kZymKg7TE

**Comment by mf-code** [comment]
[6011] - N3MYpCpCaEnM

**Comment by mf-code** [comment]
[6011] - zsEsn8y2syhm

**Comment by mf-code** [comment]
[6011] - fJ9lO5fKDoNQ

**Comment by mf-code** [comment]
[6011] - oUJ2c1auVW3S

**Comment by mf-code** [comment]
[6011] - ePRHQh8qZSYI

**Comment by mf-code** [comment]
[6011] - JHMosDlbewWb

**Comment by mf-code** [comment]
[6011] - nteUXvNjv6Xx

**Comment by mf-code** [comment]
[6011] - 5exgx0Zu4Kve

**Comment by mf-code** [comment]
[6011] - gQgYjA74VQl9

**Comment by mf-code** [comment]
[6011] - dc8U24UZxzke

**Comment by mf-code** [comment]
[6011] - 8MWqP5seEt3l

**Comment by mf-code** [comment]
[6011] - PWQMoNBaoFFE

**Comment by mf-code** [comment]
[6011] - q15KCE7VWDTU

**Comment by mf-code** [comment]
[6011] - kjXgFWx4Jn2p

**Comment by mf-code** [comment]
[6011] - openjdk8 - Allure report on Netlify: https://5f50be2f971af700c4715b01--musing-swartz-6fc060.netlify.app

**Comment by gbuh** [comment]
Sadly, I do not see any changes here regarding my comments: https://github.com/jdi-testing/jdi-light/pull/2264

**Comment by mf-code** [comment]
[6015] -

**Comment by mf-code** [comment]
[6015] -

**Comment by mf-code** [comment]
[6015] -

**Comment by mf-code** [comment]
[6015] -

**Comment by mf-code** [comment]
[6015] - YDTxG9F4kJTJ

**Comment by mf-code** [comment]
[6015] - E0ci12dHN3KT

**Comment by mf-code** [comment]
[6015] - a1ZO5yftcmd0

**Comment by mf-code** [comment]
[6015] - CRDUe5KQX2O8

**Comment by mf-code** [comment]
[6015] - 77O5vvXx9RAK

**Comment by mf-code** [comment]
[6015] - 0zI8EG43mDLN

**Comment by mf-code** [comment]
[6015] - zsaQvF1cCtmt

**Comment by mf-code** [comment]
[6015] - YzHWMbDzRj9a

**Comment by mf-code** [comment]
[6015] - jZPTFU6AyIrb

**Comment by mf-code** [comment]
[6015] - 7Ke34diTZpwn

**Comment by mf-code** [comment]
[6015] - i5SuZPPlgrPH

**Comment by mf-code** [comment]
[6015] - 10Q4nTZ6sHLK

**Comment by mf-code** [comment]
[6015] - openjdk8 - Allure report on Netlify: https://5f5108b279871143ef7973da--musing-swartz-6fc060.netlify.app

**Review by gbuh — APPROVED** [review]

**Pull request merged**